meta-box/meta-box-template.php

```diff
@@ -163,6 +163,13 @@
 			</li>
 		<?php endforeach; ?>
 	</ul>
+	<?php if ( ! $publish_with_warnings ) : ?>
+		<hr />
+		<p>
+			<input type="checkbox" id="instant_articles_force_submit" <?php if ( $force_submit ) : ?>checked<?php endif; ?> />
```

Comment:
WP's `checked()` function should be used here rather than printing out 'checked'